src/components/option/option-styles.ts

```diff
@@ -6,7 +6,6 @@ import { themeSizes, themeColors } from '../../Theme'
 const { 7: variableSize, 8: titleSize } = themeSizes
 
 export const styles = StyleSheet.create({
-  // OptionContainer
   optionContainer: {
```

Comment:
These should perhaps be renamed to:
- option
- name
- value

..or something similar

Reply:
Done